src/Bridge/Symfony/DependencyInjection/Compiler/AdapterCompilerPass.php

```diff
@@ -47,4 +47,9 @@ private function isDoctrineOrmLoaded(ContainerBuilder $container): bool
     {
         return $container->has('doctrine') && $container->has('sonata.doctrine.adapter.doctrine_orm');
     }
+
+    private function isDoctrinePHPCRLoaded(ContainerBuilder $container): bool
```

Comment:
No need for an extra method IMHO

Reply:
This is similar to `isDoctrineOrmLoaded`.

Reply:
Fine then